controllers/home/views/home-rebrand.njk

```diff
@@ -0,0 +1,84 @@
+{% extends "layouts/main.njk" %}
+{% from "components/latest-updates/macro.njk" import latestUpdates %}
+{% from "components/miniature-hero/macro.njk" import miniatureHero %}
+{% from "components/social.njk" import socialLinks with context %}
+
+{% block content %}
+    <main role="main">
+        <h1 class="u-visually-hidden">{{ globalCopy.brand.title }}</h1>
```

Comment:
Not 100% sure about this, need to check screen readers. It might make more sense for the h1 to be the opening part of the intro text.

Reply:
Yeah... maybe for SEO reasons it'd be better to have the page title as the `<h1>`, too?

Reply:
(as in "Managing your Funding" or whatever, not TNLCF)